extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeStreamExtractor.java

```diff
@@ -280,29 +273,29 @@ public StreamInfoItemsCollector getRelatedStreams() throws IOException, Extracti
 
     @Nonnull
     @Override
-    public String getSupportInfo() throws ParsingException {
+    public String getSupportInfo() {
         try {
             return JsonUtils.getString(json, "support");
         } catch (ParsingException e) {
             return "";
         }
     }
 
-    private String getRelatedStreamsUrl(List<String> tags) throws UnsupportedEncodingException {
-        String url = baseUrl + PeertubeSearchQueryHandlerFactory.SEARCH_ENDPOINT;
-        StringBuilder params = new StringBuilder();
+    private String getRelatedStreamsUrl(final List<String> tags) throws UnsupportedEncodingException {
+        final String url = baseUrl + PeertubeSearchQueryHandlerFactory.SEARCH_ENDPOINT;
+        final StringBuilder params = new StringBuilder();
         params.append("start=0&count=8&sort=-createdAt");
-        for (String tag : tags) {
+        for (final String tag : tags) {
             params.append("&tagsOneOf=");
             params.append(URLEncoder.encode(tag, "UTF-8"));
         }
         return url + "?" + params.toString();
     }
 
-    private void getStreamsFromApi(StreamInfoItemsCollector collector, String apiUrl) throws ReCaptchaException, IOException, ParsingException {
-        Response response = getDownloader().get(apiUrl);
+    private void getStreamsFromApi(final StreamInfoItemsCollector collector, final String apiUrl) throws ReCaptchaException, IOException, ParsingException {
+        final Response response = getDownloader().get(apiUrl);
         JsonObject relatedVideosJson = null;
-        if (null != response && !Utils.isBlank(response.responseBody())) {
+        if (response != null && !Utils.isBlank(response.responseBody())) {
```

Comment:
I see this is used in many places; what about adding an overload to `Utils.isNullOrEmpty()` taking a `Response`? For another PR though

Reply:
No, create isNullOrBlank, because we don't check nullness here

Reply:
Oh yeah, sorry

Reply:
Good idea, but not in this PR due to lack of time (plus it was already the case before)